### DISCUSSION TITLE
Web DJ does not connect the user

### DISCUSSION THREAD

**Discussion by nelsonepr** [AzuraCast/AzuraCast#6968] [General]
Best regards, 
yesterday I updated the Azuracast without any problems, I carried out the entire process, but when I go to transmit through Web DJ it does not start the transmission, I have https on my domain and before the update it worked without problems, any solution? Thank you very much in advance.

**Comment by phonic2k** [comment]
It's a bug in the new version:
https://github.com/AzuraCast/AzuraCast/issues/6936

**Comment by BusterNeece** [comment]
Fixed in the latest Rolling Release.

**Comment by nelsonepr** [comment]
I still have the same problem and I have the latest update__
